### ISSUE TITLE
Font unpacking PNG issues

### ISSUE THREAD

**Issue by alexsb92** [LeonBlade/xnbcli#30]
Unpacking fonts from Content/Fonts doesn't seem to work well, where the PNG generated is messed up.. 

Example attached for SmallFont. This was on Windows 10
![SmallFont](https://user-images.githubusercontent.com/1044676/85364899-32778380-b4f2-11ea-9313-b22627b8aa58.png)

Let me know if you need anything else from me!

**Comment by LeonBlade** [comment]
Which fonts? Not all of the fonts are working currently.

**Comment by Wyzard256** [comment]
I looked into this and figured out why: the fonts are compressed with DXT3 (format 5), and xnbcli isn't correctly calling dxt-js to decompress them.  Decompression of DXT formats is done with this code in `Texture2DReader.js`:

```javascript
if (format == 4)
    data = dxt.decompress(data, width, height, dxt.kDxt1);
else if (format == 5)
    data = dxt.decompress(data, width, height, dxt.kDxt3);
else if (format == 6)
    data = dxt.decompress(data, width, height, dxt.kDxt5);
```

The problem is that `dxt.kDxt1`, `dxt.kDxt3`, and `dxt.kDxt5` are all undefined, so the decompress function doesn't know which format to use.  If you change those to `dxt.flags.DXT1`, `dxt.flags.DXT3`, and `dxt.flags.DXT5` respectively (as shown in the usage example on the [dxt-js project page](https://www.npmjs.com/package/dxt-js)), the fonts decompress correctly.

However, fixing the font decompression brings out a different problem: `TypeError: pxData.copy is not a function` from the pngjs library while writing out the images.  This is a known bug in older versions of pngjs (lukeapage/pngjs#92), and it's not clear why it only happens with the font images, but it's easy to solve: upgrade to a newer version of pngjs where the bug has been fixed.  Version 6.0.0 (the latest) works and doesn't seem to have any backward-compatibility problems.

I tested by unpacking all the Stardew (PC v1.5.4) XNB files before and after making these changes, and the results are identical except for the font images being corrected.
